sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawTask.java

```diff
@@ -99,6 +107,10 @@ public Integer getMemory() {
     return memory;
   }
 
+  public Integer getShmsize() {
+    return shmsize;
```

Comment:
Hey @kaiwalyajoshi and @kvish looks like some additional changes were needed to get things working on sdk-0570-rc1

cc @rathohit311356

Reply:
This shouldn't be required with the [rc2 release](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.0-rc2).